### PR TITLE
team.yml

### PR DESCRIPTION
**Pull request by cherman2**
Adds Chloe Herman's Profile

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -84,3 +84,14 @@
       - programming
       - video games
       - guitar
+      
+- name: Chloe Herman
```
Comment (thermokarst) [inline _data/team.yml]: Awesome! Can you move this up between my entry, and Chris' entry, that way its sorted by alphabetical order of last names (excluding @gregcaporaso).

File: _data/team.yml [hunk 2]
```diff
@@ -84,3 +84,14 @@
...
+  title: PhD Student
+  pic: ChloeHerman.png
```
Comment (thermokarst) [inline _data/team.yml]: Looks like this file hasn't been included yet in the pull request - can you make sure to `git add` and `git commit` that file, too?

**Review by thermokarst — CHANGES_REQUESTED** [review]
Thanks @cherman2! This is looking good! I have two minor requests, inline. Please let me know if you have any questions.

File: _data/team.yml (hunk 1 above)
Comment (cherman2) [reply _data/team.yml]: @thermokarst, Done. Thank you!

File: _data/team.yml (hunk 2 above)
Comment (cherman2) [reply _data/team.yml]: I think that I accidentally created two forks and committed it there. I think this fixed it. Let me know.

**Review by thermokarst — APPROVED** [review]
Looks great - thanks! Welcome aboard, @cherman2!

**Pull request merged**